.github/workflows/CI.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [macOS-latest, ubuntu-latest, windows-latest]
-        python-version: [3.7, 3.8]
+        python-version: [3.8, 3.9]
```

Comment:
conda-forge is still building for 3.6 through 3.9---should we test with the same versions?

Reply:
Good question. I don't think we need to though. We want to make sure we are supporting the latest versions and following the protocol for "Most recent 2 Python Versions." I can understand supporting the "Most Recent 2"-1 version (3.7) in this case while everyone else gets pulled up. 

We can specify in the `meta.yaml` on conda-forge to restrict to Python >= 3.8 and it just wont build the lesser versions.

We also do not actually do anything which is Python 3.6 incompatible, I think, so there's no harm yet (not like we have any Walrus in the code yet).

Reply:
Maybe we don't need the full matrix for all Python versions but can still include Ubuntu. Also there's a chance `pymbar` v4 goes `noarch` if the C extension is dropped, so conda-forge would be simplified.

Reply:
I like having the full matrix in our own CI testing since it lets us see upstream dependency issues. But I agree, if we can go `noarch` then the conda-forge build will be much simpler.